### ISSUE TITLE
How to use Cutie output masks as training data to train Cutie

### ISSUE THREAD

**Issue by panyuxin1993** [hkchengrex/Cutie#106]
I want to train Cutie with our own dataset. Overall the Cutie give us quite good segmentation, so we manually corrected errors of those predictions to get a ground truth data and hope to use them to train Cutie. Do you have any suggestions which data format should we use? Are there any easy way to transform output masks of Cutie to a format that can be used as training dataset?

**Comment by hkchengrex** [comment]
We load the data with this dataset class: https://github.com/hkchengrex/Cutie/blob/main/cutie/dataset/vos_dataset.py
It might be useful to refer to the formats of those existing datasets listed at https://github.com/hkchengrex/Cutie/blob/main/docs/TRAINING.md. Namely, the DAVIS format should be the simplest. 

The output of Cutie is typically a PNG mask, which is the same as what they use in the DAVIS dataset. So yes it can be used directly. You might want to mix those new training data with the existing ones for balance.

**Comment by panyuxin1993** [comment]
Thank you for your suggestion, I noticed that the DAVIS dataset mainly contain 1 object. I am wondering how to organize my data if I have multiple objects in one picture. Should I merge annotation masks together or give multiple annotations each representing mask for one object?

**Comment by hkchengrex** [comment]
DAVIS (2017) often contains more than one object.
